Lib/turtle.py

```diff
@@ -826,7 +826,7 @@ def textinput(self, title, prompt):
         >>> screen.textinput("NIM", "Name of first player:")
 
         """
-        return simpledialog.askstring(title, prompt)
+        return simpledialog.askstring(title, prompt, parent=self.cv)
```

Comment:
If we are using this approach, you may wish to also apply it to `numinput`.

Reply:
Good catch!